schematic/models/validate_attribute.py

```diff
@@ -1649,40 +1810,59 @@ def _run_validation_across_target_manifests(
                     missing_manifest_log=missing_manifest_log,
                     present_manifest_log=present_manifest_log,
                     repeat_manifest_log=repeat_manifest_log,
+                    target_attribute_in_manifest_list=target_attribute_in_manifest_list,
+                    target_manifest_empty=target_manifest_empty,
                 )
 
             # Concatenate target manifest columns, in a subsequent step will run cross manifest validation from
             # the current manifest
             # column values against the concatenated target column
             if "value" in rule_scope:
-                target_column = self._gather_target_columns_value(
+                (
+                    target_column,
+                    target_attribute_in_manifest_list,
+                    target_manifest_empty,
+                ) = self._gather_target_columns_value(
                     column_names=column_names,
                     target_attribute=target_attribute,
                     concatenated_target_column=target_column,
                     target_manifest=target_manifest,
+                    target_attribute_in_manifest_list=target_attribute_in_manifest_list,
+                    target_manifest_empty=target_manifest_empty,
                 )
 
-        # Store outputs according to the scope for which they are used.
-        if "set" in rule_scope:
-            validation_store = (
-                missing_manifest_log,
-                present_manifest_log,
-                repeat_manifest_log,
-            )
+        if len(target_attribute_in_manifest_list) > 0:
+            if sum(target_attribute_in_manifest_list) == 0:
+                target_attribute_in_manifest = False
+            else:
+                target_attribute_in_manifest = True
 
-        elif "value" in rule_scope:
-            # From the concatenated target column, for value scope, run validation
-            (
-                missing_values,
-                duplicated_values,
-                repeat_values,
-            ) = self._run_validation_across_targets_value(
-                manifest_col=manifest_col,
-                concatenated_target_column=target_column,
-            )
-            validation_store = (missing_values, duplicated_values, repeat_values)
+        # Check if a target manifest has been found, if so do not export validation_store
+        if not target_attribute_in_manifest:
+            return (start_time, target_attribute_in_manifest)
+        elif sum(target_manifest_empty) > 0:
+            return (start_time, "values not recorded in targets stored")
+        else:
```

Comment:
These returns don't match to return type for this method

Reply:
I updated the return types